src/extension/code-actions/MissingImplementationInClassCreator.ts

```diff
@@ -9,6 +9,10 @@ import { Logger } from '../utilities/winstonLogger';
 import { ImplementPolymorphElements, NoopCodeAction } from './CodeAction';
 import { CodeActionCreator } from './CodeActionCreator';
 
+const REGEX_GENERICS = /^(.*)<(.*)>$/;
+const REGEX_INCORRECT_IMPL = /class (['"])(.*)\1 incorrectly implements.*(['"])(.*)\3\./i;
+const REGEX_NON_ABSTRACT_IMPL = /non-abstract class (['"])(.*)\1.*implement inherited.*from class (['"])(.*)\3\./i;
```

Comment:
- `<` and `>` are not special-chars by themselves: removing the class-delimiter escapes
- Ensuring quote match the usual way (group backreference)
- Stripping useless costly `g` flags (also, when re-using the same `Regexp` instances because of extraction, they'd result in a partially-sticky matching behavior we don't want)